provider/baked/src/zerotrie.rs

```diff
@@ -55,17 +55,29 @@ pub struct Data<M: DataMarker> {
 }
 
 impl<M: DataMarker> super::DataStore<M> for Data<M> {
-    fn get(&self, id: DataIdentifierBorrowed) -> Option<&'static <M>::Yokeable> {
+    fn get(
+        &self,
+        id: DataIdentifierBorrowed,
+        attributes_prefix_match: bool,
+    ) -> Option<&'static <M>::Yokeable> {
         use writeable::Writeable;
         let mut cursor = self.trie.cursor();
         let _is_ascii = id.locale.write_to(&mut cursor);
         if !id.marker_attributes.is_empty() {
             cursor.step(ID_SEPARATOR);
             id.marker_attributes.write_to(&mut cursor).ok()?;
+            loop {
+                if let Some(v) = cursor.take_value() {
+                    break Some(v);
+                }
+                if !attributes_prefix_match || cursor.probe(0).is_none() {
```

Comment:
Thought: should we be returning the identity of the match? It requires allocating unless we return it in a fixed length buffer or tinystr... hmm, if DataMarkerAttributes were a TinyStr, we could do this more easily...

Reply:
Yeah, as long as `DataResponseMetadata` is not `Copy`, which it isn't; it already has a `DataLocale`. Good that we can add it later.